airflow/docker.go

```diff
@@ -1349,49 +1339,6 @@ var createDockerProject = func(projectName, airflowHome, envFile, buildImage, se
 	return project, err
 }
 
-var checkWebserverHealth = func(settingsFile string, envConns map[string]astrocore.EnvironmentObjectConnection, project *types.Project, composeService api.Service, airflowDockerVersion uint64, noBrowser bool, timeout time.Duration) error {
-	if config.CFG.DockerCommand.GetString() == podman {
-		err := printStatus(settingsFile, envConns, project, composeService, airflowDockerVersion, noBrowser)
-		if err != nil {
-			if !errors.Is(err, errComposeProjectRunning) {
-				return err
-			}
-		}
-	} else {
-		ctx, cancel := context.WithTimeout(context.Background(), timeout)
-		defer cancel()
-		// check if webserver is healthy for user
-		err := composeService.Events(ctx, project.Name, api.EventsOptions{
-			Services: []string{WebserverDockerContainerName}, Consumer: func(event api.Event) error {
-				marshal, err := json.Marshal(map[string]interface{}{
-					"action": event.Status,
-				})
-				if err != nil {
-					return err
-				}
-				if string(marshal) == `{"action":"health_status: healthy"}` {
-					err := printStatus(settingsFile, envConns, project, composeService, airflowDockerVersion, noBrowser)
-					if err != nil {
-						return err
-					}
-				}
-
-				return nil
-			},
-		})
-		if err != nil {
-			if errors.Is(err, context.DeadlineExceeded) {
-				fmt.Printf("\n")
-				return fmt.Errorf("there might be a problem with your project starting up. The webserver health check timed out after %s but your project will continue trying to start. Run 'astro dev logs --webserver | --scheduler' for details.\n\nTry again or use the --wait flag to increase the time out", timeout)
-			}
-			if !errors.Is(err, errComposeProjectRunning) {
-				return err
-			}
-		}
-	}
-	return nil
-}
-
```

Comment:
This implementation is being removed in favor of the new one declared in its own file below - `health_check.go`.